lib/guardian/schema.graphql

```diff
@@ -0,0 +1,15 @@
+type Guardian {
+  id: ID!
+  firstName: String!
+  middleName: String
+  lastName: String!
+  addressLine1: String!
+  addressLine2: String!
+  city: String!
+  province: Province!
+  postalCode: String!
+  phone: String!
+  relationship: String!
+  notes: String
+  applicant: Applicant!
```

Comment:
I don't think this should be included in the schema, since we don't have a field resolver for `Applicant` under `Guardian`. Also, we wouldn't be querying for that anyway.